standalone/opts.go

```diff
@@ -46,7 +46,7 @@ func WithCSS(css []byte) HandlerOption {
 }
 
 // AddJS adds a JS file to Handler, serving the supplied contents at the URI
-// "/s/<filename>" with a Content-Type of "text/javascript; charset=UTF-8". It
+// "/s/<filename>" with a ContentType-Type of "text/javascript; charset=UTF-8". It
```

Comment:
huh?

Reply:
Well fudge. I did a refactor-rename in Goland at some point (to rename something from Content to ContentType) and it foolishly did crap like this to a lot of the source folders in my project root. My bad, should have caught this myself before pushing. Will fix shortly.

Reply:
Fixed. Sorry about that.